docker-compose.override.unit_tests.yml

```diff
@@ -39,6 +39,7 @@ services:
     image: busybox:1.35.0-musl
     entrypoint: ['echo', 'skipping', 'initializer']
   mysql:
+    platform: linux/amd64
```

Comment:
How is this related to the PR?

Reply:
Again, this was introduced by accident while developing on an ARM64 Mac - apparently there is no arm64 mysql so I had to specifically request the amd64. Will remove.